test/grub2base_test.rb

```diff
@@ -27,6 +27,15 @@
 
       subject.read
     end
+
+    it "reads trusted boot configuration from sysconfig" do
+      mocked_sysconfig = ::Bootloader::Sysconfig.new(trusted_boot: true)
+      expect(::Bootloader::Sysconfig).to receive(:from_system).and_return(mocked_sysconfig)
+
+      subject.read
+
+      expect(subject.trusted_boot).to eq true
```

Comment:
NP: strictly speaking, nothing ensures this `true` is coming from `mocked_sysconfig.trusted_boot`. I would like to see a complementary case with `false` to be sure.